src/inc/quic_platform_winuser.h

```diff
@@ -846,6 +891,39 @@ CxPlatSqeInitialize(
     UNREFERENCED_PARAMETER(queue);
     CxPlatZeroMemory(sqe, sizeof(*sqe));
     sqe->Completion = completion;
+
+    if (sqe->WcpEvent == NULL) {
+        sqe->WcpEvent = CreateEvent(NULL, TRUE, FALSE, NULL);
+        CXPLAT_DBG_ASSERT(sqe->WcpEvent != NULL);
+    }
+
+    NTSTATUS status = NtCreateWaitCompletionPacket(&sqe->WaitCompletionPacket,
```

Comment:
Most SQE are created as part of `CxPlatStartDatapathIo`, and the expectation is that the networking stack will queue the completion (unless they complete inline).

I don't think we should create an event and all the related completion packet setup when we expect the sqe to be completed by the kernel and not explicitly by our code (at least for now...)